src/spatial/views/heatmapViews.py

```
@@ -0,0 +1,157 @@
+            try:
+                cls._redis_client = redis.StrictRedis(host=redis_host, port=redis_port, db=redis_db, decode_responses=True)
+                cls._redis_client.ping() # Test connection
+                print(f"Connected to Redis at {redis_host}:{redis_port}")
+            except redis.exceptions.ConnectionError as e:
+                print(f"Could not connect to Redis: {e}")
+                cls._redis_client = None # Ensure client is None if connection fails
+        return cls._redis_client
+
+    @staticmethod
+    def pm25_to_aqi(pm):
+        if np.isnan(pm): return np.nan
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)